login.py

```
@@ -4,7 +4,10 @@
 def login(username, password):
     conn = sqlite3.connect("users.db")
     cursor = conn.cursor()
-    query = f"SELECT * FROM users WHERE username='{username}' AND password='{password}'"
+    query = f"SELECT * FROM users WHERE username='Aarti' AND password='Bhosale'"
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)